src/main/java/net/schmizz/sshj/xfer/scp/SCPEngine.java

```diff
@@ -39,17 +40,31 @@
         RECURSIVE('r'),
         VERBOSE('v'),
         PRESERVE_TIMES('p'),
-        QUIET('q');
+        QUIET('q'),
+        LIMIT('l', "");
 
         private final char a;
+        private String v;
```

Comment:
Why the extra 'v' here? It is not used currently. The `LIMIT` _seems_ to use it, but as it is an enum it is always the empty string.